From: Colin Ian King <colin.king@canonical.com>
Date: Mon, 19 Mar 2018 13:33:10 +0000
Subject: rbd: fix spelling mistake: "reregisteration" -> "reregistration"
Git-commit: f6870cc9a36623d1dcb0aceade9e8a4785a4283a
Patch-mainline: v4.17-rc1
References: FATE#324714

Trivial fix to spelling mistake in rdb_warn message text.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Acked-by: Luis Henriques <lhenriques@suse.com>
---
 drivers/block/rbd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/block/rbd.c
+++ b/drivers/block/rbd.c
@@ -3887,7 +3887,7 @@ static void rbd_reregister_watch(struct
 
 	ret = rbd_dev_refresh(rbd_dev);
 	if (ret)
-		rbd_warn(rbd_dev, "reregisteration refresh failed: %d", ret);
+		rbd_warn(rbd_dev, "reregistration refresh failed: %d", ret);
 }
 
 /*
